provision-user.sh

```diff
@@ -164,12 +164,16 @@ git_clone() {
 
 configure_dotfiles() {
   echo ">>> Installing eirini-home"
+  # backing up any previous existing ssh-key
+  if [ -f $HOME/.ssh/authorized_keys ]; then 
+    KEYS="$(cat $HOME/.ssh/authorized_keys)"
+  fi
```

Comment:
What is the idea behind backing up and redtoring the authorized keys? Our provisioning scripts are not supposed to modify them.

Reply:
Ok, I see. Let's leave this one in then